packages/shared/routes/dashboard/wallet/views/Send.svelte

```diff
@@ -528,7 +527,7 @@
                 <div class="w-full block">
                     {#if selectedSendType === SEND_TYPE.INTERNAL}
                         <Dropdown
-                            value={to?.label || null}
+                            value={to.alias || null}
```

Comment:
You need to check this again + some other properties on `selectedAccount` in this file, because I think they haven't been updated

Reply:
This file is going to be deleted, just left it here to cross reference the old logic when I refactor the send app logic more into the SendFormPopup and SendConfimationPopup